### PR TITLE
Web console: Update dqt to a version that adds quotes by default

### PR DESCRIPTION
**Pull request by vogievetsky**
I carried out a referendum on the Slack channel:

image

And the results called for quoting all the references in all the auto generated queries. 

See all the updated tests.

### CODE REVIEW

File: web-console/src/helpers/spec-conversion.ts [hunk 1]
```diff
@@ -38,10 +38,10 @@ export function getSpecDatasourceName(spec: IngestionSpec): string {
 function convertFilter(filter: any): SqlExpression {
   switch (filter.type) {
     case 'selector':
-      return SqlRef.columnWithQuotes(filter.dimension).equal(filter.value);
+      return SqlRef.column(filter.dimension).equal(filter.value);
```
Comment (kfaraz) [inline web-console/src/helpers/spec-conversion.ts:41]: Why don't we want quotes for filter and partition dimensions?

**Review by kfaraz — COMMENTED** [review]
Looks good. Minor query.

File: web-console/src/helpers/spec-conversion.ts (hunk 1 above)
Comment (vogievetsky) [reply web-console/src/helpers/spec-conversion.ts:41]: In DQT 0.16.x the default behavior of `SqlRef.column` changed so `SqlRef.column` is now identical to `SqlRef.columnWithQuotes`. (This is why everything started adding quotes everywhere with relatively little code changes)

Comment (kfaraz) [reply web-console/src/helpers/spec-conversion.ts:41]: Makes sense. `druid-query.ts` seems to be using `columnWithQuotes`. Should that just use `column` too?

**Review by kfaraz — APPROVED** [review]

**Comment by vogievetsky** [comment]
That one is actually using `columnWithoutQuotes` and that is ok since it is going into the signature formatting not an actual query. I kept it the same as it was before

**Pull request merged**